node/lib/package/PackageInfo.ts

```diff
@@ -181,3 +184,25 @@ export class PackageTranslatorInfo {
      */
     public readonly onboardingProperties: { [propertyName: string]: string };
 }
+
+export class PackageOnboardingInfo
+{
+    public readonly onboardingId: string;
```

Comment:
@jasongin Are you referring to onboardingId. ?. Codeflow do not place the comments correctly.

---
In reply to: [75262878](https://github.com/openT2T/opent2t/pull/22#discussion_r75262878) [](ancestors = 75262878)